validation/src/main/scala/hmda/validation/rules/lar/validity/V250.scala

```diff
@@ -0,0 +1,14 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.{ CommonDsl, Result }
+import hmda.validation.rules.EditCheck
+
+object V250 extends EditCheck[LoanApplicationRegister] {
+
+  override def apply(lar: LoanApplicationRegister): Result = {
+    (lar.loan.amount is numeric) and (lar.loan.amount is greaterThan(0))
```

Comment:
This is interesting.... The validation specifies that we need to check if `lar.loan.amount` is a number, but this field stores an Int (which is a Number).  Is the check here necessary?